content/articles/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/index.md

```diff
@@ -0,0 +1,79 @@
+<div style="text-align: center; padding-bottom: 5%;">
+<img style="padding: 0; margin:0;" src="/engineering-education/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/raspberry21.jpg" alt="Raspberry Pi diagram"><br>
+<span style="font-size: 11px;">Figure 1: Sample Rapberry Pi <a href="https://unsplash.com/photos/raLeFIxXgDY">Raspberry Pi</a>, 2021.<span>
+</div>
+
+Materials required for this tutorial include:
+1.	Raspberry Pi.
+2.	Microphone.
+3.	Internet connection.
+
+### Prerequisites
+In this guide, it is assumed that the reader has a Raspberry Pi that has a running operating system. We will assume that the running Raspberry Pi. It should also have an active connection to the internet. Additionally, you must have access to your Pi, through whichever means you prefer. It could be through a terminal or by a virtual server such as VNC. This guide will use a Putty program to run commands on our Pi. This guide is also based on **Rasbian Jessie installed Raspberry Pi 3**. 
+
+### Overview
+This article will cover:
+- [How the private FM broadcast will work](#how-the-private-fm-broadcast-will-work)
+- [Setting up the Raspberry](#setting-up-the-raspberry)
+- [Conversion of PRi to Private FM transmitter](#conversion-of-pri-to-private-fm-transmitter)
+- [Conclusion](#conclusion)
+- [Relevant resources](#relevant-resources)  
+  
+### How the private FM broadcast will work
+To reduce electromagnetic interference, all microprocessors are fitted with a synchronous digital system. Using a spread-spectrum clock signal or what is referred to as SSCS in short form, suppression of electromagnetic interference is achieved. This signal has a frequency of between 1MHz and 250MHz which falls between the FM band. Therefore, by using a program that modulates frequency, the Pi will be tweaked to operate as a radio transmitter. This signal will then be produced via the GPIO pin four of the Pi. A short wire of around twenty centimeters can be attached to act as an antenna.
+
+### Setting up the Raspberry
+Since the Raspberry Pi has a running operating system, boot it and connect the output HDMI to a monitor. Also connect a keyboard and mouse. On accessing the Desktop, search for the network option which will enable connection to the router (internet). Next, move on to the pi configuration through the menu and enable SSH communication. 
+Once this is enabled, we can now access the Raspberry using Putty as long the personal computer and the Pi are connected to the same network. Open Putty on the personal computer and key in the IP address of the Raspberry Pi to connect via SSH. A session will then appear and ask for a login credentials. The credentials to be used here are pi for username and raspberry for password. 
+
+### Conversion of PRi to Private FM transmitter
+*Step One:*
```

Comment:
For a second I was wondering what's PRi? Please explain what PRi is in one sentence before getting started with Step one